README.md

```diff
@@ -1,157 +1,47 @@
-## Trashed
-# Keep an eye on resource usage.
+## Barnes - GC Statsd Reporter
 
-
- - Sends per-request object counts, heap growth, GC time, and more to StatsD.
- - Sends snapshots of resource usage, e.g. live String objects, to StatsD.
- - Supports new stuff: Rails 5.1 and latest Ruby 2.x features.
- - Supports old stuff: Rails 2/3/4, Ruby 1.9+, REE, Ruby 1.8 with RubyBench patches.
+A fork of [trashed](https://github.com/basecamp/trashed) focused on Ruby metrics for Heroku.
 
 ## Setup
 
 ### Rails 5
 
-On Rails 5 (and Rails 3 and 4), add this to the top of `config/application.rb`:
-
-    require 'trashed/railtie'
-
-And in the body of your app config:
-
-    module YourApp
-      class Application < Rails::Application
-        config.trashed.statsd = YourApp.statsd
-
-
-### Rails 2
-
-On Rails 2, add the middleware to `config/environment.rb`:
-
-    Rails::Initializer.run do |config|
-      reporter = Trashed::Reporter.new
-      reporter.logger = Rails.logger
-      reporter.statsd = YourApp.statsd
-
-      config.middleware.use Trashed::Rack, reporter
-    end
-
-
-### Custom dimensions
+On Rails 5 (and Rails 3 and 4), add this to your Gemfile:
 
-You probably want stats per controller, action, right?
-
-Set a `#timing_dimensions` lambda to return a list of dimensions to
-qualify per-request measurements like time elapsed, GC time, objects
-allocated, etc.
-
-For example:
-```ruby
-config.trashed.timing_dimensions = ->(env) do
-  # Rails 3, 4, and 5, set this. Other Rack endpoints won't have it.
-  if controller = env['action_controller.instance']
-    name    = controller.controller_name
-    action  = controller.action_name
-    format  = controller.rendered_format || :none
-    variant = controller.request.variant || :none  # Rails 4.1+ only!
-
-    [ :All,
-      :"Controllers.#{name}",
-      :"Actions.#{name}.#{action}.#{format}+#{variant}" ]
-  end
-end
-```
-
-Results in metrics like:
 ```
-YourNamespace.All.Time.wall
-YourNamespace.Controllers.SessionsController.Time.wall
-YourNamespace.Actions.SessionsController.index.json+phone.Time.wall
+gem "barnes"
 ```
 
+Then run:
 
-Similarly, set a `#gauge_dimensions` lambda to return a list of dimensions to
-qualify measurements which gauge current state, like heap slots used or total
-number of live String objects.
-
-For example:
-
-```ruby
-config.trashed.gauge_dimensions = ->(env) {
-  [ :All,
-    :"Stage.#{Rails.env}",
-    :"Hosts.#{`hostname -s`.chomp}" ]
-}
-```
-
-Results in metrics like:
 ```
-YourNamespace.All.Objects.T_STRING
-YourNamespace.Stage.production.Objects.T_STRING
-YourNamespace.Hosts.host-001.Objects.T_STRING
+$ bundle install
 ```
 
+### Non-Rails
 
-### Version history
```

Comment:
It sucks to lose the version history. Any appetite to start fresh with barnes 0.0.1?

Reply:
Would prefer a CHANGELOG file. I can add one.

Reply:
Added